ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml

```diff
@@ -40,6 +40,15 @@
                         components="[group:hclecomm.Content]">
                         <jcr:content jcr:primaryType="nt:unstructured"/>
                     </policy_600116016309972>
+		    <policy_414135893823493
```

Comment:
@Rajamouli-hcl : where is this policy being used in which template ?

Reply:
reused content-ploicy in experience-fragment web variation template and  removed old policy which i have created previously